extensions/src/libadalang-helpers.ads

```diff
@@ -76,8 +76,15 @@ package Libadalang.Helpers is
    --  Try to create a unit provider out of ``Project``. If not possible, call
    --  ``Abort_App``.
 
+   function Command_Line_Event_Handler
+     (Exit_On_Missing_File : Boolean) return Event_Handler_Reference;
+   --  Create an event handler with default callbacks for command line
+   --  applications. ``Exit_On_Missing_File`` will determine the behavior when
+   --  encountering a missing dependency. If ``False``, a warning will be shown
+   --  but resolution will continue. If ``True``, application will exit.
```

Comment:
Can you please specify that 1) an error is emitted when exitting 2) both warnings and errors are emitted on the standard error stream?